plugins/GSdx/GSSetting.cpp

```diff
@@ -55,7 +55,9 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_ALPHAHACK:
 			return "Different alpha handling. Can work around some shadow problems.";
 		case IDC_OFFSETHACK:
-			return "Might fix some misaligned fog, bloom, or blend effect.";
+			return "Might fix some misaligned fog, bloom, or blend effect."
+				"Special cases are only uselful in couple of games like Captain Tsubasa."
+				"Special cases don't work for now.";
```

Comment:
Special cases aren't working because Gregory originally used additional variable and ComboBox on Linux but want old and new offset hack merged now.